tasks/build/download_node_builds.js

```diff
@@ -50,11 +52,9 @@ module.exports = function (grunt) {
       } else {
         await fromNode(cb => {
           resp
-          .pipe(createGunzip())
-          .on('error', cb)
-          .pipe(new Extract({ path: downloadDir, strip: 1 }))
-          .on('error', cb)
-          .on('end', cb);
+            .pipe(createWriteStream(resolve(downloadDir, platform.nodeUrl.substring(platform.nodeUrl.lastIndexOf('/') + 1))))
+            .on('error', cb)
+            .on('finish', cb);
```

Comment:
exactly. aftert the file is downloaded i check shasum and only after that i can extract the archive